src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,14 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> {
+                    int personAge = person.getAge();
+                    return person.getSex() == Person.Sex.MAN
+                            ? personAge >= fromAge && person.getAge() <= maleToAge
+                            : personAge >= fromAge && person.getAge() <= femaleToAge;
+                })
```

Comment:
personAge >= fromAge is common for both
```suggestion
                    return personAge >= fromAge && (person.getSex() == Person.Sex.MAN 
                            ? person.getAge() <= maleToAge : person.getAge() <= femaleToAge);
                })
```